disnake/abc.py

```diff
@@ -1535,6 +1540,16 @@ async def send(
 
             .. versionadded:: 2.5
 
+        flags: Optional[:class:`.MessageFlags`]
```

Comment:
why is this documented as Optional? None of the other parameters to this method are documented as optional, because of a disclaimer at the beginning of the docstring.

> At least one of ``content``, ``embed``/``embeds``, ``file``/``files``,
        ``stickers``, ``components``, or ``view`` must be provided.

Reply:
Fair point, changed in https://github.com/DisnakeDev/disnake/pull/929/commits/442da80be82b18ebc9f833b571504ad4e954d9b8.